cdap-ui/app/services/constants.js

```diff
@@ -49,7 +49,8 @@ angular.module(PKG.name + '.services')
           unconnectedNodesError: 'Please connect all nodes.',
           circularConnectionError: 'Please remove the circular connection in this pipeline.',
           endSinkError: 'Please end the pipeline connections in a sink.',
-          parallelConnectionError: 'Please remove parallel connections in this pipeline.'
+          parallelConnectionError: 'Please remove parallel connections in this pipeline.',
+          pluginDoesNotExist: ' plugin does not exist'
```

Comment:
Inconsistent error message; it has a lowercase sentence beginning and no period at the end.